starknet_py/compile/compiler.py

```diff
@@ -27,7 +31,7 @@ class Compiler:
 
     def __init__(
         self,
-        contract_source: List[StarknetCompilationSource],
+        contract_source: StarknetCompilationSource,
```

Comment:
First `StarknetCompilationSource = NewType(
    "CairoSource", Union[CairoSourceCode, List[CairoFilename]]
)` 
Also current compiler doesn't seem to work correctly with list of `CairoSources`. When i changed `test_create_definition` in `compiler_test.py` to pass `[test_file_content]` instead of just `test_file_content` I was getting errors. We either need to change compiler somehow so it accepts multiple CairoSourceCodes (don't know if it's possible), or change typing like I did here.

Reply:
I think I noticed it somewhere long time ago, @Arcticae do you confirm a list doesn't work? If so we should just remove it from the type.

Reply:
Hmm i think the intent was here to provide the interface which can do 2 things:

- Provide a list of input files 
- Provide a snippet of compilable cairo, for demonstration purposes and quick compiling. 

I don't think list of loaded source codes is viable in the implementation - i'll take a look

Reply:
Yep that's right
```

def load_source_code(
    src: List[StarknetCompilationSource],
) -> List[Tuple[str, str]]:
    if isinstance(src, str):
        return [(src, str(hash(src)))]
    return [(load_cairo_source_code(filename), filename) for filename in src]
```

Reply:
So it's correct that I changed it to just `StarknetCompilationSource` without the `List`?